src/msw/printdlg.cpp

```diff
@@ -852,14 +857,16 @@ bool wxWindowsPrintDialog::ConvertToNative( wxPrintDialogData &data )
         (wxWindowsPrintNativeData *) data.GetPrintData().GetNativeData();
     data.GetPrintData().ConvertToNative();
 
-    PRINTDLG *pd = (PRINTDLG*) m_printDlg;
+    PRINTDLGEX* pd = (PRINTDLGEX*) m_printDlg;
 
     // Shouldn't have been defined anywhere
     if (pd)
         return false;
 
-    pd = new PRINTDLG;
-    memset( pd, 0, sizeof(PRINTDLG) );
+    pd = (LPPRINTDLGEX) GlobalAlloc(GPTR, sizeof(PRINTDLGEX));
```

Comment:
I don't really mind doing this, but I just wonder why did you decide to change this, i.e. switch from `new` to `GlobalAlloc` here? There doesn't seem to be any reason to use the Windows function as we only free this one ourselves, so why?

Reply:
Ok, changed.